README.md

```diff
@@ -4,45 +4,43 @@
-- [版本发行日志](https://github.com/sofastack/sofa-jraft/wiki/%E7%89%88%E6%9C%AC%E5%8F%91%E8%A1%8C%E6%97%A5%E5%BF%97)
-
-## 如何贡献
-[如何参与 SOFAJRaft 代码贡献](https://github.com/sofastack/sofa-jraft/wiki/%E5%A6%82%E4%BD%95%E5%8F%82%E4%B8%8E-SOFAJRaft-%E4%BB%A3%E7%A0%81%E8%B4%A1%E7%8C%AE)
-
-## 致谢
-SOFAJRaft 是从百度的 [braft](https://github.com/brpc/braft) 移植而来，做了一些优化和改进，感谢百度 braft 团队开源了如此优秀的 C++ RAFT 实现
-
-## 开源许可
-SOFAJRaft 基于 [Apache License 2.0](./LICENSE) 协议，SOFAJRaft 依赖了一些第三方组件，它们的开源协议也为 Apache License 2.0，
-另外 SOFAJRaft 也直接引用了一些开源协议为 Apache License 2.0 的代码（可能有一些小小的改动）包括：
-- [JCTools](https://github.com/JCTools/JCTools) 中的 NonBlockingHashMap/NonBlockingHashMapLong
-- [Netty](https://github.com/netty/netty) 中的 HashedWheelTimer，另外还参考了 Netty 的 Pipeline 设计
-- [Protobuf](https://github.com/protocolbuffers/protobuf) 中对 UTF8 String 高效的编码/解码
-
-
+## Overview
+SOFAJRaft is a production-level, high-performance Java implementation based on the [RAFT](https://raft.github.io/) consistency algorithm that supports MULTI-RAFT-GROUP for high-load, low-latency scenarios.
+With SOFAJRaft you can focus on your business area. SOFAJRaft handles all RAFT-related technical challenges. SOFAJRaft is very easy to use, which you can master in a short amount of time with a few examples.
+
+## Features
+- Leader election
+- Log replication and recovery
+- Snapshot and log compaction
+- Cluster production membership management, adding nodes, deleting nodes, replacing nodes, etc.
+- Proactively change Leader for restart maintenance, Leader load balancing, etc.
+- Symmetric network partition tolerance
+- Asymmetric network partition tolerance
+- Fault tolerance, minority failure doesn't affect the overall availability of system
+- Manual recovery cluster available for majority failure
+- Linearizable read, ReadIndex/LeaseRead
+- Replication pipeline optimistic
+- Built-in performance metrics based on [Metrics](https://metrics.dropwizard.io/4.0.0/getting-started.html) class library, with rich performance statistics
+- Passed [Jepsen](https://github.com/jepsen-io/jepsen) consistency verification test
+- JRaft includes an embedded distributed KV implementation
+
+## Requirements
+Build-time requirement: JDK 8 or above and Maven 3.2.5 or above.
+
+## Documents
+- [User Guide](https://github.com/sofastack/sofa-jraft/wiki)
+- [Counter Example Details](https://github.com/sofastack/sofa-jraft/wiki/Counter-%E4%BE%8B%E5%AD%90%E8%AF%A6%E8%A7%A3)
+- [Release Notes](https://github.com/sofastack/sofa-jraft/wiki/%E7%89%88%E6%9C%AC%E5%8F%91%E8%A1%8C%E6%97%A5%E5%BF%97)
+
+## Contribution
+[How to contribute](https://github.com/sofastack/sofa-jraft/wiki/%E5%A6%82%E4%BD%95%E5%8F%82%E4%B8%8E-SOFAJRaft-%E4%BB%A3%E7%A0%81%E8%B4%A1%E7%8C%AE)
+
+## Acknowledgement
+SOFAJRaft was ported from Baidu's [braft](https://github.com/brpc/braft) and made some optimizations and improvements. Thanks to the Baidu braft team for opening up such a great C++ RAFT implementation.
```

Comment:
"SOFAJRaft was ported from Baidu's [braft](https://github.com/brpc/braft) and made some optimizations and improvements." -> "SOFAJRaft was ported from Baidu's [braft](https://github.com/brpc/braft) with some optimizing and improvement."